src/Documentation/DocumentationGenerator.php

```diff
@@ -450,18 +450,28 @@ public function generateRuleSetsDocumentationIndex(array $setDefinitions)
     }
 
     /**
-     * @param int    $sampleIndex
+     * @param int    $sampleNumber
```

Comment:
i let myself fix it and solve the conflicts with 2.15 branch

Reply:
What needed to be fixed?

Reply:
the diff is the fix - you forgot to rename var name in docs ;)